### PR TITLE
Fix for concourse version on statefulset for workers picking up TSA host

### LINKED ISSUE #4902: Concourse CI workers failing on kubernetes
**Is this a request for help?**:
Yes
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
Helm: v2.9.0-rc3
Kubernetes: v1.9.3-gke.0

**Which chart**:
stable/concourse

**What happened**:
worker pods keeps failing, with the error message: 
failed to connect to TSA: dial tcp: address jaunty-magpie-web: missing port in address

**What you expected to happen**:
Workers running and i could run the command fly workers and see the workers.

**How to reproduce it** (as minimally and precisely as possible):
helm install stable/concourse, and when everything is running, the workers starts failing.

**Anything else we need to know**:
I have tried the default values file, and with a nginx ingress controller.

### PR DESCRIPTION
**Pull request by brunoban**
**What this PR does / why we need it**:
Concourse workers are currently not being able to communicate to TSA due to a problem with the environment variable CONCOURSE_TSA_HOST. It's being checked against version 3.10 instead of 3.10.x.

**Which issue this PR fixes**:

fixes #4902 #4812 

**Special notes for your reviewer**:
@frodenas
@william-tran 
Not sure if this is the best way to solve this, but it seemed to work with my cluster.

### CODE REVIEW

File: stable/concourse/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: concourse
-version: 1.2.3
+version: 1.2.4
```
Comment (william-tran) [inline stable/concourse/Chart.yaml]: bump the patch version to 1.3.1

**Review by william-tran — CHANGES_REQUESTED** [review]
@brunoban Thanks for the fix, please update the chart version to bump the patch version to 1.3.1 and I can merge this in.

**Comment by brunoban** [comment]
Done! Thanks @william-tran

**Pull request merged**